.travis.yml

```diff
@@ -1,42 +0,0 @@
-language: python
-install:
-  - pip install tox
-  - python setup.py install_egg_info
-script:
-  - tox
-matrix:
-  include:
-    - python: 3.5
-      env: TOXENV=py35-1.11.X
-    - python: 3.6
-      env: TOXENV=py36-1.11.X
-    - python: 3.7
-      env: TOXENV=py37-1.11.X
-      dist: xenial
-    - python: 3.8
-      env: TOXENV=py38-1.11.X
-      dist: xenial
-    - python: 3.5
-      env: TOXENV=py35-2.2.X
-    - python: 3.6
-      env: TOXENV=py36-2.2.X
-    - python: 3.7
-      env: TOXENV=py37-2.2.X
-      dist: xenial
-    - python: 3.8
-      env: TOXENV=py38-2.2.X
-      dist: xenial
-    - python: 3.6
-      env: TOXENV=py36-3.0.X
-    - python: 3.7
-      env: TOXENV=py37-3.0.X
-      dist: xenial
-    - python: 3.8
-      env: TOXENV=py38-3.0.X
-      dist: xenial
-
-notifications:
-  irc: "irc.freenode.org#django-compressor"
```

Comment:
Not sure how to replicate that, but there is an action which might help: https://github.com/marketplace/actions/notify-irc

Reply:
This can be dropped